known-issues.yml

```diff
@@ -421,6 +429,15 @@ zib-TextResult.VisualResult:
     Media.content:
       - cardinality: 1..1
         reason: The use of the Media resource referenced in zib-TextResult DiagnosticReport.media.link is optional (0..*), but if the Media resource is used, Media.content should be present (1..1).
+        
+zib-TreatmentObjective:
+  zib deviations:
+    Goal.note.text:
+      - cardinality: 1..1
+        reason: The mapped element has a cardinality of 1..1. However, the parent element, .note, has a cardinality of 0..1, effectively resulting in a cardinality of 0..1 for this element.
+    Goal.target:
+      - datatype: BackboneElement
+        reason: The zib FunctionalOrMentalStatus is mapped to an Observation resource, which represents something that has been observed in the past. The zib TreatmentObjective describes a desired result in the future. Using a reference to a zib-FunctionalOrMentalStatus instance of Observation would therefore be incorrect. This is why the properties of FunctionalOrMentalStatus have been mapped in this zib-TreatmentObjective profile.
```

Comment:
I think this description misses the point of what's going on. zib FunctionalOrMentalStatus exists as an Observation resource because that's most fitting when using it in the past, but for the current context we need to have a resource that describes a future situation, and _that_ is not available as a separate resource.

> Proposal for a better text: Although the zib model assumes there's a reference from zib OutcomeOfCare to a distinct instance of zib FunctionalOrMentalStatus (in future context), in this profile the concepts of zib FunctionalOrMentalStatus are mapped directly as this aligns with the way the Goal resource is used.

Also: we should consider to rename the zib-FunctionalOrMentalStatus profile on Observation to zib-FunctionalOrMentalStatus-event

Reply:
BTW. this comment should also be adjusted in the profile.

Reply:
Thnx Pieter. Adjusted!